administrator/components/com_menus/models/item.php

```diff
@@ -1273,7 +1273,8 @@ public function save($data)
 		$dispatcher = JEventDispatcher::getInstance();
 		$pk         = (!empty($data['id'])) ? $data['id'] : (int) $this->getState('item.id');
 		$isNew      = true;
-		$table      = $this->getTable();
+		JModelLegacy::addTablePath(JPATH_ADMINISTRATOR . '/components/com_menus/tables');
```

Comment:
This is wrong code. There is also a way to fix this. You don't really needs to change this code. I would suggest refer https://api.joomla.org/cms-3/classes/JModelLegacy.html and you will get idea what you needs to do in article controller.